src/python/correct_fastq.py

```diff
@@ -39,41 +39,44 @@ def get_barcodes(whitelist_file):
     return list(r1_barcodes), list(r2_barcodes), list(r3_barcodes)
 
 
-def create_barcode_dict(barcode_list):
+def create_barcode_dicts(barcode_list):
```

Comment:
This function is used here and in raw_bam_to_fastq. That means that we need to always make sure that if we change one we will change also the other.  The best thing would be to put them in a util file and source it here.